user/user_test.go

```diff
@@ -45,7 +45,7 @@ func (md *MockDAO) UpdateUser(userID int64, request dao.UserUpdateRequest) (*dao
 			md.Users[i].Name = request.Name
 			return &dao.UserUpdateResponse{
 				ID:   user.ID,
-				Name: user.Name,
+				Name: request.Name,
```

Comment:
We been using `req` or `request` elsewhere? Just wondering for generation

Reply:
https://github.com/TempleEight/spec-golang/blob/642f55d12c21fdb7ac77c0f820905c6a4d63da00/user/dao/dao.go#L79

`request`